rules/windows/defense_evasion_encoding_or_decoding_files_via_certutil.toml

```diff
@@ -25,7 +30,7 @@ type = "eql"
 query = '''
 process where event.type == "start" and
   (process.name : "certutil.exe" or process.pe.original_file_name == "CertUtil.exe") and 
-  process.args : ("?decode", "?encode")
+  process.args : ("?decode", "?encode", "?urlcache", "?verifyctl", "?decodehex")
```

Comment:
```suggestion
  process.args : ("?decode", "?encode", "?encodehex", "?decodehex")
```

Reply:
I like the `urlcache` and `verifyctl` commands, but they don't deal with encoding/decoding. I think they could be split into a separate rule, or we modify this one into something like `Suspicious CertUtil Commands`.  If we choose the latter, I will adjust my review.

Reply:
@bm11100 yeah I think changing it to `Suspicious CertUtil Commands` would be better then adding another rule. 

I'm guessing to make these additional changes?

```
name = "Suspicious CertUtil Commands"
```

File: `defense_evasion_suspicious_certutil_commands.toml`

Reply:
@bm11100 how would you like me to proceed.